packages/dependency-extraction-webpack-plugin/package.json

```
@@ -0,0 +1,32 @@
+{
+	"name": "@wordpress/dependency-extraction-webpack-plugin",
+	"version": "1.0.0",
```

```suggestion
	"version": "1.0.0+alpha.0",
```
so we could leave room for version bump for Lerna